dbms/src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -277,6 +277,13 @@ StorageReplicatedMergeTree::StorageReplicatedMergeTree(
         return;
     }
 
+    if (attach && !current_zookeeper->exists(zookeeper_path + "/metadata"))
```

Comment:
Let us also handle the case when the metadata only for the single replica was deleted from ZK.

Reply:
Decided not to do.